WeatherApp/WeatherApp/Base.lproj/Main.storyboard

```diff
@@ -16,29 +16,97 @@
+                            <constraint firstItem="cPI-el-zKr" firstAttribute="top" secondItem="swK-HI-aRQ" secondAttribute="bottom" constant="40" id="YGw-07-XF1"/>
+                            <constraint firstItem="OVy-tR-IiX" firstAttribute="centerY" secondItem="cPI-el-zKr" secondAttribute="centerY" id="bso-9L-UOl"/>
+                            <constraint firstItem="6Tk-OE-BBY" firstAttribute="trailing" secondItem="OVy-tR-IiX" secondAttribute="trailing" constant="20" id="kar-yI-K4u"/>
+                            <constraint firstItem="swK-HI-aRQ" firstAttribute="top" secondItem="fQ2-Pu-QFe" secondAttribute="bottom" constant="89" id="lyi-bh-7AB"/>
+                            <constraint firstItem="IeF-10-HRp" firstAttribute="top" secondItem="fQ2-Pu-QFe" secondAttribute="bottom" constant="89" id="nXG-Ne-xe8"/>
+                            <constraint firstItem="6Tk-OE-BBY" firstAttribute="bottom" secondItem="eZG-n0-8MY" secondAttribute="bottom" constant="152" id="pUr-Qm-GAi"/>
+                            <constraint firstItem="swK-HI-aRQ" firstAttribute="leading" secondItem="6Tk-OE-BBY" secondAttribute="leading" constant="20" id="pyf-Yk-Gnb"/>
+                            <constraint firstItem="6Tk-OE-BBY" firstAttribute="trailing" secondItem="IeF-10-HRp" secondAttribute="trailing" constant="20" id="sfF-Gk-PLm"/>
+                            <constraint firstItem="6Tk-OE-BBY" firstAttribute="trailing" secondItem="fQ2-Pu-QFe" secondAttribute="trailing" constant="20" id="vdU-mw-cpR"/>
+                            <constraint firstItem="fQ2-Pu-QFe" firstAttribute="top" secondItem="6Tk-OE-BBY" secondAttribute="top" constant="107" id="wzp-H1-Qra"/>
+                        </constraints>
                     </view>
+                    <connections>
+                        <outlet property="latTextField" destination="swK-HI-aRQ" id="cDQ-gX-GNC"/>
+                        <outlet property="lonTextField" destination="cPI-el-zKr" id="Pbq-Jw-Rzp"/>
+                        <outlet property="weatherLabel" destination="fQ2-Pu-QFe" id="nQs-hP-Qzf"/>
+                    </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="dkx-z0-nzr" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="-197" y="54"/>
+            <point key="canvasLocation" x="-197.10144927536234" y="53.571428571428569"/>
         </scene>
     </scenes>
     <resources>
         <systemColor name="systemBackgroundColor">
             <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
         </systemColor>
-        <systemColor name="systemGreenColor">
-            <color red="0.20392156862745098" green="0.7803921568627451" blue="0.34901960784313724" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
-        </systemColor>
     </resources>
 </document>
```

Comment:
add viewController on window in appdelegate. We will still have controller in storyboard  but we need to set up it manually. also remove entry point in storyboard